virttest/utils_test/libvirt.py

```diff
@@ -1044,6 +1045,16 @@ def pre_vol(self, vol_name, vol_format, capacity, allocation, pool_name):
         if not pv.volume_exists(vol_name):
             raise exceptions.TestFail("Can't find volume: %s" % vol_name)
 
+    def pre_vol_xml(self, pool_name, **vol_params):
```

Comment:
This function create a volume, but from the name, it more like create a XML file, so I suggest you can change the name to 'pre_vol_by_xml'